scout/server/app.py

```diff
@@ -35,6 +35,40 @@
 )
 
 LOG = logging.getLogger(__name__)
+USERS_LOGGER_PATH_PARAM = "USERS_ACTIVITY_LOG_PATH"
```

Comment:
😂 These no repeated strings things make the python code read like assembler!

Reply:
Is that good or bad?

Reply:
Ha, good question! I kind of liked assembler, especially compared to the "high" level languages like BASIC that were the early alternatives for me. But I have not felt the need to go back to that style since I learned C and C++. 😂